addOns/paramminer/src/main/javahelp/org/zaproxy/addon/paramminer/resources/help/contents/dialog.html

```diff
@@ -0,0 +1,65 @@
+<!DOCTYPE html PUBLIC "-//W3C//DTD HTML 3.2 Final//EN">
+<HTML>
+<HEAD>
+<META HTTP-EQUIV="Content-Type" CONTENT="text/html; charset=utf-8">
+<TITLE>
+Param Miner dialog
+</TITLE>
+</HEAD>
+<BODY>
+<H1>Param Miner dialog</H1>
+<p>
+This dialog launches the <a href="paramminer.html">spider</a>.<br>
+</p>
+
+<H2>Control</H2>
+The first tab allows you to select or change the URL along with selection of guessers.<br>
+If the URL is in one or more Contexts then you will be able to choose one of them.<br>
+You can also modify the Threadpool size that would be used for by the paraminer. <br> <br>
```

Comment:
- `paraminer`  → `Param Miner`
 - `Threadpool size`  → `number of threads`, the UI should be changed accordingly.

Reply:
It was the number of threads before ... we decided and changed that :)

Reply:
I don't recall it was.